hummingbot/strategy/order_tracker.pyx

```diff
@@ -307,3 +310,9 @@ cdef class OrderTracker(TimeIterator):
 
     def remove_create_order_pending(self, order_id: str):
         self.c_remove_create_order_pending(order_id)
+
+    async def all_orders_being_cancelled_confirmation(self, market_info: MarketTradingPairTuple):
```

Comment:
Can this be done by simply checking if the in_flight_cancels list is empty?
These wait tasks can make it difficult to debug and at least it should have time out accompanied.